domain/src/main/java/com/architect/codes/mu8/interactor/comics/IComicUseCase.kt

```diff
@@ -0,0 +1,7 @@
+package com.architect.codes.mu8.interactor.comics
+
+import com.architect.codes.mu8.model.Comic
+
+interface IComicUseCase {
```

Comment:
Rename as `ComicUseCase`, also you can put this interface in the same file as the implementation to avoid more files since they are on the same package.

Reply:
done